_docs/other/discord-bot-usage.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: page
+title: Discord Bot Usage
+permalink: /other/discord-bot-usage/
+---
+
+### Our discord bot provides some very useful toos to debug/issue track your server. We will go in-depth on how to use the bot commands and its tools.
+
+## Server Logs
+
+If you are having console errors or Geyser isn't functioning/starting up, server logs are very useful to find the root cause of why Geyser is not running. You can safely share your server logs with this website; [mclogs](https://mclo.gs) as this website will remove all IP-Adresses from the logs. if you paste the logs url into or discord our bot will analyze the error and will give an fix if there is one present like seen below. You can also paste the lastest.log file into discord.
```

Comment:
```suggestion
If you are having console errors or Geyser isn't functioning/starting up, server logs are very useful to find the root cause of why Geyser is not running. You can safely share your server logs with [mclogs](https://mclo.gs) as it will remove all IP addresses and other sensitive information from the logs. 

If you paste the logs URL into our Discord, our bot will analyze the error and will give a fix if there is one present like seen below. You can also paste the `latest.log` file into Discord.
```